sql/2020/08_Accessibility/common_element_attributes.sql

```diff
@@ -0,0 +1,37 @@
+#standardSQL
+# How often pages contain an element with a given attribute
+CREATE TEMPORARY FUNCTION getUsedAttributes(payload STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+try {
+  const almanac = JSON.parse(payload);
+  return Object.keys(almanac.attributes_used_on_elements);
+} catch (e) {
+  return [];
+}
+''';
+SELECT
+  _TABLE_SUFFIX AS client,
+  total_sites,
+  attribute,
+  COUNT(0) AS total_sites_using,
+  COUNT(0) / total_sites AS pct_sites_using
+FROM
+  `httparchive.pages.2020_08_01_*`,
+  UNNEST(getUsedAttributes(JSON_EXTRACT_SCALAR(payload, '$._almanac'))) AS attribute
+LEFT JOIN (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(0) AS total_sites
+  FROM
+    `httparchive.pages.2020_08_01_*`
+  GROUP BY _TABLE_SUFFIX
+)
+USING (_TABLE_SUFFIX)
+GROUP BY
+  client,
+  attribute,
+  total_sites
+HAVING
+  total_sites_using > 1000
```

Comment:
I don't think this threshold is high enough - there are 16,000 rows in the sheet. There could be a lot of unusable data in there, making it harder to sift through and it slows down the sheet's performance.

\>= 1% still leaves 1,000 results

Also consider filtering the attributes to a known set of a11y attributes.
```suggestion
  pct_sites_using >= 0.01
```

Reply:
The tough part is this data isn't just for `aria-*` properties, and some of aria properties like `aria-grabbed` only have 0.00018952% usage.

If we're worried about sheet performance, I could remove rows Q-Z in the sheet. So we'd reduce the number of cells by half. Or I could also manually delete non-aria properties until we reach 0.01% of usage like you're suggesting

Reply:
Maybe something like this
```suggestion
  STARTS_WITH(attribute, 'aria-') OR
  total_sites_using > 1000
```

Reply:
LGTM